libyul/backends/wasm/WordSizeTransform.cpp

```diff
@@ -96,12 +96,12 @@ void WordSizeTransform::operator()(Block& _block)
 							for (int i = 0; i < 3; i++)
 								ret.push_back(VariableDeclaration{
 									varDecl.location,
-									{TypedName{varDecl.location, newLhs[i], "u64"_yulstring}},
```

Comment:
Hmm, these used `u64` but now you're using `i64`.

Reply:
Yes, because there actually is no `u64` in wasm.